client/src/modules/report-group/index.html

```diff
@@ -0,0 +1,170 @@
+    <h4> {{"New reporting group"}}</h4>
+
+
+    <form
+      name="ReportGroupRegistrationForm"
+      bh-submit="ReportGroupCtrl.save(ReportGroupRegistrationForm)"
+      bh-form-defaults
+      novalidate>
+      <!-- code -->
+      <div
+      class="form-group"
+      ng-class="{'has-error' : ReportGroupRegistrationForm.code.$invalid && ReportGroupRegistrationForm.$submitted}">
+
+      <label class="control-label ng-binding">
+        Code
+      </label>
+
+      <div class="input-group col-lg-12">
+        <input
+        class="form-control"
+        name="code"
+        ng-model="ReportGroupCtrl.reportGroup.code"
+        autocomplete="off"
+        required>
+
+        <div class="help-block"
+        ng-messages="ReportGroupRegistrationForm.code.$error"
+        ng-show="ReportGroupRegistrationForm.$submitted">
+        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+        </div>
+      </div>
+      </div>
+      <!-- name -->
+      <div
+      class="form-group"
+      ng-class="{'has-error' : ReportGroupRegistrationForm.name.$invalid && ReportGroupRegistrationForm.$submitted}">
+
+      <label class="control-label ng-binding">
+        Name
+      </label>
+
+      <div class="input-group col-lg-12">
+        <input class="form-control"
+        type="text"
+        name="name"
+        ng-model="ReportGroupCtrl.reportGroup.name"
+        autocomplete="off"
+        required/>
+        <div class="help-block"
+        ng-messages="ReportGroupRegistrationForm.name.$error"
+        ng-show="ReportGroupRegistrationForm.$submitted">
+
+        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+        </div>
+      </div>
+      </div>
+      <!-- description -->
+
+      <div
+      class="form-group"
+      ng-class="{'has-error' : ReportGroupRegistrationForm.description.$invalid && ReportGroupRegistrationForm.$submitted}">
+
+      <label class="control-label ng-binding">
+        Description
+      </label>
+
+      <div class="input-group col-lg-12">
+        <textarea
+        class="form-control"
+        name="description"
+        ng-model="ReportGroupCtrl.reportGroup.description"j
+        autocomplete="off"
+        required>
+        </textarea>
+        <div class="help-block"
+        ng-messages="ReportGroupRegistrationForm.description.$error"
+        ng-show="ReportGroupRegistrationForm.$submitted">
+          <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+        </div>
+      </div>
+      </div>
+
+
+      <bh-loading-button
+      ng-show="ReportGroupCtrl.selectedReportGroup.selected == false"
+            class="pull-right" loading-state="BedRegistrationForm.$loading">
+              {{ "Save"}}
+          </bh-loading-button>
+
+          <span
+            class="btn btn-danger"
+            ng-click="ReportGroupCtrl.remove(ReportGroupRegistrationForm)"
+            ng-show="ReportGroupCtrl.selectedReportGroup.selected == true"
```

Comment:
There is no need to compare to `true` here.  You can just write:
```html
ng-show="ReportGroupCtrl.selectedReportGroup.selected"
```
with the same effect.